### ISSUE TITLE
Blocking cookies/referrer breaks embedded private Vimeo players (restricted to certain websites)

### ISSUE THREAD

**Issue by ndarville** [EFForg/privacybadger#188]
![screen shot 2014-06-02 at 11 32 32](https://cloud.githubusercontent.com/assets/198682/3145572/0c3aa63c-ea39-11e3-8969-0302ab68f4d5.png)

Privacy Badger makes the video “private”. Whitelisting Vimeo in Privacy Badger fixed the problem.

The edge case here is probably that the video has been set up to only allow embedding on an explicit list of domains, which interferes with Privacy Badger’s protection.

**Comment by alexandru** [comment]
Dropping a note in support of fixing this — Vimeo is being used a lot for videos behind paywalls, private videos, being in general a cheap hosting solution that isn't YouTube.

I noticed people on forums complaining about videos not working on certain diet-related websites, unaware that it's Privacy Badger that breaks those websites.

And because the videos are kept private, unlisted on Vimeo for Vimeo visitors, there's no way to give them a link to those videos.

**Comment by ghostwords** [comment]
Stripping the referrer header also breaks some (?) `akamaihd.net`-powered videos. This came up before as #1969.

For a current example, see https://www.nasa.gov/multimedia/nasatv/#public where `nasa-i.akamaihd.net` needs to be "green" (not "yellow") for the video to work.

**Comment by ghostwords** [comment]
We have lots from error reports but most of them are for online courses gated by logins. I found three for the [Vimeo replacement widget PR](https://github.com/EFForg/privacybadger/pull/2266), I'm sure there are others there if you go back farther/query smarter.

**Comment by bcyphers** [comment]
Ah missed those, thanks!

**Comment by alexandru** [comment]
> Does anyone have a current example of a site with a Vimeo embed that PB breaks?

I've got 2 videos posted here:

- https://monix.io/blog/2018/11/07/tutorial-bracket.html
- https://monix.io/blog/2018/11/08/tutorial-future-to-task.html

**Comment by shapiro125** [comment]
Because this issue isn't closed yet, I'd rather post here than create a new issue. The new Vimeo replacement widget is breaking autoplay background Vimeo videos for me (Firefox 65/Chrome 72, MacOS 10.14.3, Privacy Badger 2019.1.30).

Two examples: 

- https://brand.duke.edu/
- http://knpdesign.co/

Happy to open a new issue if preferred.

**Comment by ghostwords** [comment]
Yes, could you please open a new issue? Here is one more: https://www.bethelu.edu/

**Comment by bcyphers** [comment]
This also appears to affect some embedded YouTube players.

Another broken streaming plugin: vidible.tv https://www.huffingtonpost.com/entry/amy-klobuchar-abuse-staff-2020_us_5c5a1cb1e4b0871047588649?j7

**Comment by Anutrix** [comment]
[vodhls-vh.akamaihd.net](vodhls-vh.akamaihd.net
) is blocked at [Gaana Music Streaming Site](gaana.com) which prevents songs from playing/streaming.
Making slider green fixes the issue almost instantly. 
Issue seems similar because akamahd.net was mentioned her before. Hence didn't make a new issue.

**Comment by ghostwords** [comment]
Hi @Anutrix, would you mind opening a new issue? These issues are related but are not the same.

**Comment by Anutrix** [comment]
@ghostwords Created new issue #2405

**Comment by ghostwords** [comment]
Restored Vimeo to the yellowlist (3e5833485529591bc20838393bead0f74b5e72df) for now.
